### PR TITLE
Update development.rst

### PR DESCRIPTION
**Pull request by jeffblack360**
Someone on IRC reported the issue.

### CODE REVIEW

File: fr/bake/development.rst [hunk 1]
```diff
@@ -191,7 +191,7 @@ un nouveau template de bake, faîtes ce qui suit:
    Bake
 #. Créez un nouveau répertoire **plugins/[name]/src/Template/Bake/Template/**.
 #. Copiez tout template que vous souhaitez changer à partir de
-   **vendor/cakephp/cakephp/src/Template/Bake/Template/** vers les
+   **vendor/cakephp/src/Template/Bake/Template/** vers les
```
Comment (bcrowe) [inline fr/bake/development.rst]: This should be vendor/cakephp/**bake**/src/Template/Bake/Template

**Comment by bcrowe** [comment]
Already merged & fixed in: https://github.com/cakephp/docs/pull/3394

**Comment by jeffblack360** [comment]
Thanks for the correction @bcrowe.

**Comment by bcrowe** [comment]
No problem. Keep up the good work!

**Pull request closed without merging**